readthedocs/core/static-src/core/js/doc-embed/footer.js

```diff
@@ -92,23 +92,6 @@ function injectFooter(data) {
     }
 }
 
-
-function setupBookmarkCSRFToken() {
```

Comment:
Since we are touching this .js file, I think we need to call `gulp build` and add the file generated (I'm not sure though).

In any case, if you don't know how to do it, let me know and I can check this by myself before merging: http://docs.readthedocs.io/en/latest/development/standards.html#getting-started

Reply:
Correct, you'll need to regenerate the built assets with this PR.